src/config.nim

```diff
@@ -36,6 +36,8 @@ macro parseObject(obj: typed, key, val: string) =
 proc loadConfig*(filePath: string): Config =
   result = defaultConfig
   for rawLine in filePath.lines:
+    if rawLine.len == 0:
+      continue
```

Comment:
This doesn't fully address the root cause of the bug. Non-empty lines consisting of only whitespaces also crash the application.

Reply:
Indeed. I do not know nim so that's all I could do. Maybe if you check `pair.len == 0` after https://github.com/tsoding/boomer/blob/8e8f37fa0a222c807d35f5192bedbdc2f4851b65/src/config.nim#L42 ? And in case it is indeed 0, you `continue`

Reply:
Why do you think `pair.len == 0` in case of an empty string?

Reply:
Doesn't matter. I solved it after all, should I make a new pull request?